packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -313,13 +332,15 @@ const BaseButton = ({
 
   return (
     <StyledBaseButton
+      {...makeAccessible({ role: 'button', label: accessibilityLabel })}
```

Comment:
Probably we should enforce or show warning if only icon is present but no accessibilityLabel is given.

Reply:
Discussed, we will expose `accessibilityLabel`